src/main/java/org/apache/commons/collections4/IterableUtils.java

```diff
@@ -203,11 +203,11 @@ public static <E> Iterable<E> chainedIterable(final Iterable<? extends E> a,
      * @throws NullPointerException if either of the provided iterables is null
      */
     @SuppressWarnings("unchecked")
-    public static <E> Iterable<E> chainedIterable(final Iterable<? extends E> a,
-                                                  final Iterable<? extends E> b,
-                                                  final Iterable<? extends E> c,
-                                                  final Iterable<? extends E> d) {
-        return chainedIterable(new Iterable[] {a, b, c, d});
+    public static <E> Iterable<E> chainedIterable(final Iterable<? extends E> e,
+                                                  final Iterable<? extends E> f,
+                                                  final Iterable<? extends E> g,
+                                                  final Iterable<? extends E> h ) {
+        return chainedIterable();
```

Comment:
It is unclear what this PR is for, but this is a bug as the args are now ignored.